### PR TITLE
Blog/2016-07-21-ouce-speaker-talks

### PR DESCRIPTION
**Pull request by indigo423**

### CODE REVIEW

File: user/pages/05.blog/2016-07-21-ouce-speaker-talks/item.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+body_classes: header-lite fullwidth blogstyling
+taxonomy:
+    category: events
+    tag: [ouce, events]
+---
+
+# OUCE 2016 - Speaker and Talks
+
+OpenNMS User Conference: Europe (OUCE) is a series of talks about monitoring and network management with OpenNMS.
+Our conference creates a time and place for the OpenNMS community to share infomration, discuss ideas, and work together to improve monitoring with the free OpenNMS software.
```
Comment (mvrueden) [inline user/pages/05.blog/2016-07-21-ouce-speaker-talks/item.md]: `information` vs `infomration`

File: user/pages/05.blog/2016-07-21-ouce-speaker-talks/item.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+This years OUCE 2016 will take place at the University of Applied Science in Fulda, Germany.
+
+You can attend the conference to get information about the latest tools and technologies used in monitoring and share your experience with other users and core contributors of OpenNMS.
+The three day conference begins on 13 September with a training from David Hustace, who will explain all the possibilities of visualizing monitoring information using the new Business Service Monitoring capabilities and building performance data dashboards with Grafana.
```
Comment (mvrueden) [inline user/pages/05.blog/2016-07-21-ouce-speaker-talks/item.md]: `13` vs `13th`?

File: user/pages/05.blog/2016-07-21-ouce-speaker-talks/item.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+On 14 and 15 September, our speakers give talks about the latest information and usecases for OpenNMS. Developers and contributors will also give talks about the new features and technologies currently being developed.
```
Comment (mvrueden) [inline user/pages/05.blog/2016-07-21-ouce-speaker-talks/item.md]: `14 and 15` vs `14th and 15th`

**Comment by indigo423** [comment]
👍  @mvrueden

**Pull request merged**